images/linux/scripts/installers/git.sh

```diff
@@ -46,7 +46,15 @@ DocumentInstalledItem "Git Large File Storage (LFS) ($(git-lfs --version 2>&1 |
 DocumentInstalledItem "Git-ftp ($(git-ftp --version | cut -d ' ' -f 3))"
 
 #Install hub
-snap install hub --classic
+TEMP_HUB=$(mktemp -d -t)
+pushd "$TEMP_HUB" || exit
+URL=$(curl -s https://api.github.com/repos/github/hub/releases/latest | jq -r '.assets[].browser_download_url | select(contains("hub-linux-amd64"))')
+wget -P "$TEMP_HUB" "$URL"
```

Comment:
Could you please use [download_with_retries](https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/helpers/install.sh#L7) here? This is a preferred way to download something that allows to increase image generation stability.

Reply:
Done